docs/README.md

```diff
@@ -1,5 +1,10 @@
 # docs
-Here we document important concepts and functionality formica supports.
+Here we document important concepts and functionality Inago supports.
 
 ### table of contents
-- [slice expansion](slice_expansion.md)
+
+- [What is the goal of Inago?](goal.md)
+- [What do we expect from the users?](expect.md)
```

Comment:
> Unit file content doesn't need to be in a certain format. 

Yes, they need to be. There must be a folder. There must be files within the folder. The files must start with the foldername. If you want slices, there must be `@` at the end of the basename. It is recommended(!) to have the `MachineOf=`, `Requires=` and `After=` options in the files. The file MUST be a systemd unitfile. 

All of this is the format we expect. If the user breaks that, stuff won't work